### PR TITLE
Initial release notes for 1.8.4

### PR DESCRIPTION
**Pull request by jyaistMap**
*   composed based on PRs merged into the maintenance branch and public repo as of 2020-01-13 11:00am PST
* @priyankatuteja Can you review how I've phrased the notes for those items pertaining to the `arcgis.learn` module and let me know if I need to change any

-----

# Checklist

Please go through each entry in the below checklist and mark an 'X' if that condition has been met. Every entry should be marked with an 'X' to be get the Pull Request approved.

- [ ] All `import`s are in the first cell? First block of imports are standard libraries, second block are 3rd party libraries, third block are all `arcgis` imports? Note that in some cases, for samples, it is a good idea to keep the imports next to where they are used, particularly for uncommonly used features that we want to highlight.
- [ ] All `GIS` object instantiations are one of the following?
    - `gis = GIS()`
    - `gis = GIS('https://www.arcgis.com', 'arcgis_python', 'P@ssword123')`
    - `gis = GIS(profile="your_online_profile")`
    - `gis = GIS('https://pythonapi.playground.esri.com/portal', 'arcgis_python', 'amazing_arcgis_123')`
    - `gis = GIS(profile="your_enterprise_portal")`
- [ ] If this notebook requires setup or teardown, did you add the appropriate code to `./misc/setup.py` and/or `./misc/teardown.py`?
- [ ] If this notebook references any portal items that need to be staged on AGOL/Python API playground, did you coordinate with a Python API team member to stage the item the correct way with the api\_data\_owner user?
- [ ] Code refactored & split out across multiple cells, useful comments?
- [ ] Consistent voice/tense/narrative style? Thoroughly checked for typos?
- [ ] All images used like `` instead of ``? All map widgets contain a static image preview? (Call `mapview_inst.take_screenshot()` to do so)
- [ ] All file paths are constructed in an OS-agnostic fashion with `os.path.join()`? (Instead of `r"\foo\bar"`, `os.path.join(os.path.sep, "foo", "bar")`, etc.)
- [ ] **IF YOU WANT THIS SAMPLE TO BE DISPLAYED ON THE DEVELOPERS.ARCGIS.COM WEBSITE**, ping @ DavidJVitale so he can add it to the list for the next deploy

### CODE REVIEW

File: guide/02-api-overview/release-notes.ipynb [hunk 1]
```diff
@@ -1,5 +1,91 @@
 {
```
Comment (priyankatuteja) [inline guide/02-api-overview/release-notes.ipynb:1]: <p><a href="https://developers.arcgis.com/python/api-reference/arcgis.learn.toc.html#image-translation-models" target="_blank">Image Translation Models</a></p><ul><li><code><a href="https://developers.arcgis.com/python/api-reference/arcgis.learn.toc.html#cyclegan" target="_blank">CycleGAN</a></code></li></ul><p> </p><p>The similar bug is fixed in PixToPix model also. </p><p>PixToPix also had metrics missing https://github.com/ArcGIS/geosaurus/pull/5569. </p><p> </p><p>Rest of the notes look good. Thanks!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/902/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (jyaistMap) [reply guide/02-api-overview/release-notes.ipynb:1]: Thanks @priyankatuteja 
* added the information you provided to the release notes

**Comment by jyaistMap** [comment]
@AtmaMani 
* good to go with all PRs closed on the milestone

**Review by AtmaMani — APPROVED** [review]

**Comment by AtmaMani** [comment]
thank you @jyaistMap

**Pull request merged**